Backend/ServiceLayer/Services/PointService.cs

```diff
@@ -17,7 +18,15 @@ public PointService()
 
         public Point CreatePoint(DatabaseContext _db, Point point)
         {
-            return _PointRepo.CreatePoint(_db, point);
+            try
+            {
+                var pointCreated = _PointRepo.CreatePoint(_db, point);
+                return pointCreated;
+            }
+            catch(ArgumentOutOfRangeException e)
+            {
+                throw new InvalidPointException("Longitude/Latitude value invalid.", e);
+            }

```

Comment:
Catches the exception from the lower layer and wraps it to the point specific error to make error handling easier.